dnf/repo.py

```diff
@@ -886,7 +886,9 @@ def load(self):
 
         """
         if self.metadata or self._try_cache():
-            if self._check_config_file_age and self.repofile \
+            if self.metadata_expire != -1 \
```

Comment:
yeah, you are right (about `>0`).. @mhatina could you fix documentation please? also, what about `never`?